SU-HSDO/suhumsci
# Shs 104 megamenu animation

## Summary
This adds transitions effects to the new megamenu. 

## Steps to Test
1. Visit a site with the new mega menu enabled. 
2. At desktop, hover over the top level menu items. The underline should animate.
3. Click a menu item with a dropdown. The child menu container should animate. 

**Notes**
* This is pointing to the release branch to avoid a bunch of files below. When there's a new sprint branch, we can change that.
* I did this ticket out of order. Other tickets weren't ready, and I felt I could get this one done with the info it had.
* This would be _slightly_ better if I transitioned the translate Y axis, but that would involve a lot of refactoring to 'hide' it behind everything else.
